### PR TITLE
VEP: clearly document that any species/assembly can be run in CLI (e111)

### PR DESCRIPTION
**Pull request by nuno-agostinho**
[ENSVAR-5748](https://www.ebi.ac.uk/panda/jira/browse/ENSVAR-5748)

Test in my sandbox: http://wp-np2-11.ebi.ac.uk:6070/info/docs/tools/vep/index.html

### CODE REVIEW

File: docs/htdocs/info/docs/tools/vep/script/vep_cache.html [hunk 1]
```diff
@@ -589,15 +599,19 @@ <h2 id="fasta">FASTA files</h2>
 
   <p> Suitable reference FASTA files are available to download from the Ensembl
   FTP server. See the <a href="/info/data/ftp/index.html">Downloads</a> page
-  for details.<br />You should preferably use the installer as described above to
+  for details. <p/>
```
Comment (jgtate) [inline docs/htdocs/info/docs/tools/vep/script/vep_cache.html:602]:
```suggestion
  for details. </p>
```

File: docs/htdocs/info/docs/tools/vep/script/vep_cache.html [hunk 2]
```diff
@@ -589,15 +599,19 @@ <h2 id="fasta">FASTA files</h2>
...
+  <p>You should preferably use the installer as described above to
   fetch these files; manual instructions are provided for reference.
   In most cases it is best to download the single large
   "primary_assembly" file for your species. You should use the unmasked
-  (without "_rm" or "_sm" in the name) sequences.<br />Note that VEP requires
-  that the file be either unzipped (Bio::DB::Fasta) or unzipped and then
-  recompressed with bgzip (Bio::DB::HTS::Faidx) to run; when unzipped these files can be very
-  large (25GB for human). An example set of commands for setting up the data
-  for human follows: </p>
+  (without <kbd>_rm</kbd> or <kbd>_sm</kbd> in the name) sequences. <p/>
```
Comment (jgtate) [inline docs/htdocs/info/docs/tools/vep/script/vep_cache.html:608]:
```suggestion
  (without <kbd>_rm</kbd> or <kbd>_sm</kbd> in the name) sequences. </p>
```

File: docs/htdocs/info/docs/tools/vep/script/vep_cache.html [hunk 3]
```diff
@@ -341,25 +347,23 @@ <h2 id="gff">GFF/GTF files</h2>
 
   <ul>
     <li>
-      <p><b>GFF file</b><br />Example of command line with GFF, using of flag <a href="vep_options.html#opt_gff">--gff</a> :</p>
+      <p><b>GFF file</b><p/>
```
Comment (jgtate) [inline docs/htdocs/info/docs/tools/vep/script/vep_cache.html:350]:
```suggestion
      <p><b>GFF file</b></p>
```

File: docs/htdocs/info/docs/tools/vep/script/vep_cache.html [hunk 4]
```diff
@@ -341,25 +347,23 @@ <h2 id="gff">GFF/GTF files</h2>
...
-      <pre class="code sh_sh">--custom file=data.gff.gz,format=gff</pre>
+      <p><b>NOTE:</b> If you wish to customise the name of the GFF as it appears in the SOURCE field and VEP output header, use the <a href="vep_custom.html#custom_options">longer <kbd>--custom</kbd> annotation form</a>:</p>
 
-      <p><b>NOTE:</b> You should use the <a href="vep_custom.html#custom_options">longer custom annotation form</a> if you wish to customise the name of the GFF as it appears in the SOURCE field and VEP output header.</p>
+      <pre class="code sh_sh">--custom file=data.gff.gz,short_name=frequency,format=gff</pre>
     </li>
 
     <li style="margin-top:2em">
-      <p><b>GTF file</b><br />Example of command line with GTF, using of flag <a href="vep_options.html#opt_gtf">--gtf</a> :</p>
+      <p><b>GTF file</b><p/>
```
Comment (jgtate) [inline docs/htdocs/info/docs/tools/vep/script/vep_cache.html:360]:
```suggestion
      <p><b>GTF file</b></p>
```

File: docs/htdocs/info/docs/tools/vep/script/vep_cache.html [hunk 5]
```diff
@@ -38,7 +38,13 @@ <h1 id="top"><span style="color:#006;padding-right:15px">Variant Effect Predicto
 
   <ul>
     <li><b><a href="#cache">Cache</a></b> - a downloadable file containing all transcript models, regulatory features and variant data for a species</li>
-    <li><b><a href="#gff">GFF or GTF</a></b> - use transcript models defined in a tabix-indexed GFF or GTF file</li>
+    <li><b><a href="#gff">GFF or GTF</a></b> - use transcript models defined in a tabix-indexed GFF or GTF file
+      <ul><li>
+        Requires a <b><a href="vep_cache.html#fasta">FASTA</a></b> file in
+        offline mode or if the desired species or assembly is not part of the
```
Comment (nakib103) [inline docs/htdocs/info/docs/tools/vep/script/vep_cache.html:44]: A link to `--offline` mode is better here - 

```suggestion
        <a href="vep_options.html#opt_offline">--offline</a> mode or if the desired species or assembly is not part of the
```

**Review by nakib103 — CHANGES_REQUESTED** [review]

**Review by nakib103 — APPROVED** [review]
Looks good, thanks Nuno!

**Pull request merged**